libs/test-commons/src/libs/test-commons.ts

```diff
@@ -9,3 +11,12 @@ export const shouldRunOnlyOn = (impl: string[], current: string) => impl.include
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 //@ts-ignore
 export const testIfSupportedOperationsIncludes = (supportedOperations: string[], operation: string[]): any => operation.every((o: any) => supportedOperations.includes(o)) ? test : test.skip 
+
+export const testSupportedOperations = (supportedOperations: string[], arrayTable: any[][]): string[][] => {
```

Comment:
Maybe instead of string[] 
replace it to: `supportedOperations: SchemaOperations[]`